### ISSUE TITLE
Allow dropping files into the database file name text box

### ISSUE THREAD

**Issue by bricelam** [bricelam/VS.Data.Sqlite#13]

**Comment by bricelam** [comment]
Weird. It doesn't work. I'm gonna need someone smarter than me to explain why it's not working and say whether it can be made to. Failed attempt in the [feature/drop](https://github.com/bricelam/VS.Data.Sqlite/compare/feature/drop) branch.

**Comment by ErikEJ** [comment]
Maybe you need to handle DragOver also? https://stackoverflow.com/questions/9464001/drag-and-drop-file-into-textbox

**Comment by bricelam** [comment]
Nope, still didn't work. :( My best guess is that the UserControl is being hosted inside a native window and that's somehow throwing off the event routing.

**Comment by ErikEJ** [comment]
So basically not possible. Wonder if @reduckted has any bright ideas?
